lib/erlef_web/controllers/members/slack_invite_controller.ex

```diff
@@ -36,8 +36,9 @@ defmodule ErlefWeb.Members.SlackInviteController do
 
   defp success_msg do
     """
-    Your slack invite request was successfully created.  
-    An administrator will process your request as soon as possible. 
+    Your slack invite request was successfully created. 
+    Note: There are humans behind this process and as such it can take up 
+    to 24 hours to process an invite. 
```

Comment:
```suggestion
    Your Slack invitation request was successfully created. 
    Note: there are humans behind this process and as such it can take up 
    to 24 hours to process it. 
```

Reply:
Will commit this suggestion and then see what it looks like. Definitely like the language better.

Reply:
<img width="1415" alt="Screen Shot 2021-03-15 at 8 48 01 PM" src="https://user-images.githubusercontent.com/39971740/111243891-ced93d00-85cf-11eb-98f1-967a02d61743.png">


Looks good 👍